src/Plugin/Field/FieldFormatter/EmbeddableFormatter.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+namespace Drupal\stanford_media\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\media\Entity\MediaType;
+use Drupal\Core\Messenger\MessengerInterface;
+use Drupal\media\OEmbed\ResourceFetcherInterface;
+use Drupal\media\OEmbed\UrlResolverInterface;
+use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+use Drupal\stanford_media\Plugin\media\Source\Embeddable;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\media\IFrameUrlHelper;
+use Drupal\media\Plugin\Field\FieldFormatter\OEmbedFormatter;
+
+/**
+ * Field formatter for embeddables.
+ *
+ * @FieldFormatter (
+ *   id = "embeddable_formatter",
+ *   label = @Translation("Embeddable field formatter"),
+ *   description = @Translation("Field formatter for Embeddable media."),
+ *   field_types = {
+ *     "link",
+ *     "string",
+ *     "string_long",
+ *   }
+ * )
+ */
+class EmbeddableFormatter extends OEmbedFormatter {
+
+  /**
+   * The name of the oEmbed field.
+   *
+   * @var string
+   */
+  protected $oEmbedField;
+
+  /**
+   * {@inheritDoc}
+   */
+  public function __construct($plugin_id, $plugin_definition, FieldDefinitionInterface $field_definition, array $settings, $label, $view_mode, array $third_party_settings, MessengerInterface $messenger, ResourceFetcherInterface $resource_fetcher, UrlResolverInterface $url_resolver, LoggerChannelFactoryInterface $logger_factory, ConfigFactoryInterface $config_factory, IFrameUrlHelper $iframe_url_helper) {
+
+    parent::__construct($plugin_id, $plugin_definition, $field_definition, $settings, $label, $view_mode, $third_party_settings, $messenger, $resource_fetcher, $url_resolver, $logger_factory, $config_factory, $iframe_url_helper);
+
+    $this->oEmbedField = $config_factory->get('media.type.embeddable')->get('source_configuration.oembed_field_name');
```

Comment:
the media type name is not reliable. also don't use config. use the entity itself. you can see the `isApplicable` method doing the same type of thing.